controllers/APILogic/games.js

```diff
@@ -13,28 +13,25 @@ const createFilter = (req) => {
         }
     }
 
-    if (req.query.minPlayers) {
-        filters.minPlayers = req.query.minPlayers
-    }
-
-    if (req.query.maxPlayers) {
-        filters.maxPlayers = req.query.maxPlayers
+    if (req.query.numPlayers) {
```

Comment:
not sure what the $lte and $gte business is coming from, but I'll default to you knowing what you're doing on that one ha.

Reply:
:)  this is mongoose logic for doing the database query. gte = greater than or equal, you get the gist for lte then.